test/cypress/support/index.ts

```diff
@@ -0,0 +1,169 @@
+  { first, last, address, city, postcode, phone, hint },
+  isBilling = true,
+) => {
+  const fieldPrefix = isBilling ? '#billing_' : '#shipping_'
+  const selectPrefix = isBilling ? '#select2-billing_' : '#select2-shipping_'
+  cy.intercept(/api.what3words.com\/v3\/autosuggest/i).as('autosuggest')
+    .get(`${fieldPrefix}first_name`).focus().clear().type(first)
+    .get(`${fieldPrefix}last_name`).focus().clear().type(last)
+    .get(`${fieldPrefix}address_1`).focus().clear().type(address)
+    .get(`${fieldPrefix}city`).focus().clear().type(city)
+    .get(`${fieldPrefix}postcode`).focus().clear().type(postcode)
+    .get(`${selectPrefix}country-container`).click()
+    .get('li').contains('United Kingdom').click()
+  
+  if (isBilling) cy.get(`${fieldPrefix}phone`).focus().clear().type(phone)
+  
+  cy.get(isBilling ? '#w3w-billing' : '#w3w-shipping').click().clear().type(hint)
+    .wait('@autosuggest')
+    .get(isBilling ? '#w3w-billing' : '#w3w-shipping')
+      .closest('what3words-autosuggest')
+      .find('[data-testid=suggestion-0]').click()
+})
+
+Cypress.Commands.add('saveSettings', () => {
+  cy.getElementByDataTestId('save_settings').click({ force: true })
+})
+
+Cypress.Commands.add('saveAdvanced', () => {
+  cy.getElementByDataTestId('save_advanced').click({ force: true })
+})
+
+Cypress.Commands.add('placeOrder', () => {
+  cy.get('#place_order').click()
+})
+
+Cypress.Commands.add('selectManagedInput', () => {
+  cy.getElementByDataTestId('enable_w3w_managed').click({ force: true })
+})
+
+Cypress.Commands.add('selectInputSelector', () => {
+  cy.getElementByDataTestId('enable_input_selector').click({ force: true })
+})
+
+Cypress.Commands.add('toggleReturnCoordinates', () => {
+  cy.getElementByDataTestId('return_coordinates').click({ force: true })
+})
+
+Cypress.Commands.add('toggleNearestPlace', () => {
+  cy.getElementByDataTestId('save_nearest_place').click({ force: true })
+})
+
+Cypress.Commands.add('toggleLabel', () => {
+  cy.getElementByDataTestId('enable_label').click({ force: true })
+})
+
+Cypress.Commands.add('toggleAdvancedSettings', () => {
+  cy.getElementByDataTestId('advanced_dropdown').click({ force: true })
+})
+
+Cypress.Commands.add('togglePlaceholder', () => {
+  cy.getElementByDataTestId('enable_placeholder').click({ force: true })
+})
+
+Cypress.Commands.add('toggleClipToCountry', () => {
+  cy.getElementByDataTestId('enable_clip_to_country').click({ force: true })
+})
+
+Cypress.Commands.add('toggleClipToCircle', () => {
+  cy.getElementByDataTestId('enable_clip_to_circle').click({ force: true })
+})
+
+Cypress.Commands.add('toggleClipToBoundingBox', () => {
+  cy.getElementByDataTestId('enable_clip_to_bounding_box').click({ force: true })
+})
+
+Cypress.Commands.add('toggleClipToPolygon', () => {
+  cy.getElementByDataTestId('enable_clip_to_polygon').click({ force: true })
+})
+
+Cypress.Commands.add('saveAdvanced', () => {
+  cy.getElementByDataTestId('save_advanced').click({ force: true })
```

Comment:
All these `force: true`s are making me wary as they could mask problems. Why are they needed here/can they be removed?

Reply:
This is necessary because WordPress 🙄  layers a div on top of everything so Cypress complains that it is not clickable without `force: true` unfortunately. Annoying but necessary unfortunately.

Reply:
@c5haw exactly in most cases that's the issue, I would suggest prepending `.should('be.visible')` to make sure we don't force click a hidden element

Reply:
Good shout @GHermet will add that also, cheers.